CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/svg/SVGGraphBuilder.java

```diff
@@ -673,11 +671,11 @@ private void buildConnections(final SVGObject svgGraph) throws InterruptedExcept
                             }
                             
                             // The low arrow head is not in view 
-                            break;
+                        }
 
                         // Undirected connections are Transactions, Edges and Links with no arrow heads.
-                        default:
-                            break;
+                        default  -> {
+                        }
```

Comment:
Sonar wants a comment inside an empty default block to confirm it is intentionally empty.
ie.
// Do nothing
or any other comment message more suited to the context.